packages/interactive_media_ads/android/build.gradle

```diff
@@ -54,7 +54,7 @@ android {
         testImplementation 'org.jetbrains.kotlin:kotlin-test'
         testImplementation "org.mockito.kotlin:mockito-kotlin:5.4.0"
         testImplementation 'org.mockito:mockito-inline:5.1.0'
-        testImplementation 'androidx.test:core:1.3.0'
```

Comment:
FYI I filed https://github.com/flutter/flutter/issues/164025 to track making these updates easier.